app/(site)/events/[eventPage]/page.tsx

```diff
@@ -1,91 +1,86 @@
-"use client"; // Use client to render the page 
+"use client"; // Use client to render the page
 
 import { getEventPage } from "@/backend/sanity-utils";
-import { EventPageType } from "@/types/EventPageType";
-import {PortableText} from '@portabletext/react'
-import Footer from "@/components/footer/footer";
 import BackButton from "@/components/UI/backbutton";
 import RegistrerButton from "@/components/UI/registrerbutton";
-import { useEffect, useState } from "react";
-import { redirect } from 'next/navigation';
-import getDateTimeFormat from "@/utils/date";
+import Footer from "@/components/footer/footer";
 import LoadingPage from "@/components/loadingPage/loadingPage";
+import { EventPageType } from "@/types/EventPageType";
+import getDateTimeFormat from "@/utils/date";
+import { PortableText } from "@portabletext/react";
 import { error } from "console";
-import { useRouter } from 'next/navigation'
+import { redirect, useRouter } from "next/navigation";
+import { useEffect, useState } from "react";
 
-// Props for the event page 
+// Props for the event page
 type Props = {
-  params: { eventPage: string }
-}
+  params: { eventPage: string };
+};
 
-export default function PageForEvent({params}: Props) {
+export default function PageForEvent({ params }: Props) {
   const [eventPage, setEventPage] = useState<EventPageType | null>(null);
 
   const slug = params.eventPage;
-  const router = useRouter()
+  const router = useRouter();
 
```

Comment:
@Mosazghi  Do you have a linter that adds semicolon at the end of each file?

Reply:
Yes i do.

Reply:
Could we add this linter to the project in another PR? So that everyone can use it? I belive you can define it in the eslint file in the project

Reply:
You can create another issue for this, and if you want, create a PR for it too 💯 @Mosazghi

Reply:
Will do 💪.